src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,34 @@
 package core.basesyntax;
 
 public class MainApp {
+    public MainApp(String[] machine) {
 
+        Machine truck = new Truck() {
+            @Override
+
+            public void doWork() {
+                super.doWork();
+            }
```

Comment:
There's no need to override the `doWork` method without changing its behavior. If you're not adding additional functionality, you can directly create an instance of `Truck` without the anonymous class.